src/components/Header/components/PeopleSearch/people-search.scss

```diff
@@ -4,8 +4,9 @@
   .people-search-dropdown {
     position: absolute;
     max-height: 60vh;
-    overflow-y: auto;
+    overflow-y: auto; // make this to scroll
```

Comment:
wording awkward--is this a TODO? Or is it a note about why the line is necessary?